src/Microsoft.ML.Data/Transforms/ConcatEstimator.cs

```diff
@@ -123,90 +112,6 @@ public SchemaShape GetOutputSchema(SchemaShape inputSchema)
         }
     }
 
-    // REVIEW: Note that the presence of this thing is a temporary measure only.
```

Comment:
>// REVIEW: Note that the presence of this thing is a temporary measure only. [](start = 4, length = 76)

People spend time writing this code, and you deleting it without any remorse!

Reply:
Extra credit for removing Tom's code!

---
In reply to: [217230777](https://github.com/dotnet/machinelearning/pull/896#discussion_r217230777) [](ancestors = 217230777)

Reply:
If a code falls in the forest...

---
In reply to: [217551182](https://github.com/dotnet/machinelearning/pull/896#discussion_r217551182) [](ancestors = 217551182,217230777)